javascript/lib/api/src/model/things.model.ts

```diff
@@ -63,8 +63,8 @@ export class Thing extends EntityWithId {
   }
 
   public toObject(): object {
-    const featuresObj = Features.toObject(this.features);
-    const aclObj = Acl.toObject(this._acl);
+    const featuresObj = this.features ? Features.toObject(this.features) : undefined;
```

Comment:
On master, it's
```
const featuresObj = Features.toObject(this.features);
const aclObj = Acl.toObject(this._acl);
```
Do we need the extra check?

Reply:
Actually, no, the toObject method should be able to handle undefined